### DISCUSSION TITLE
yumbee - qa tester

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#95] [Applications] [answered]
hire?
![Screenshot from 2024-04-05 09-08-53](https://github.com/mariocraft987/bark.github.io/assets/129226914/d90e0ec2-2121-435f-befd-3dbc49f9d362)

**Comment by mariocraft987** [comment, marked as answer]
check discord

**Comment by Rubiidev-18** [comment]
does he have a github acc tho?

**Reply by mariocraft987** [reply]
I agree with rubiidev

**Reply by raynepaws** [reply]
i just asked them
